setup.py

```diff
@@ -39,15 +37,8 @@
         "opt_einsum>=2.3.2",
     ],
     extras_require={
-        "torch": [
-            "pyro-ppl>=1.5.2",
-            "torch>=1.7.0",
-        ],
-        "jax": [
-            "numpyro>=0.2.4",
-            "jax>=0.1.57",
-            "jaxlib>=0.1.37",
-        ],
```

Comment:
@fritzo Do you have a config file for black? I run `make format` but it causes a lot of changes.

Reply:
Our only config file is [setup.cfg](https://github.com/pyro-ppl/funsor/blob/master/setup.cfg) but we do not configure black. Maybe you need to update isort, whose black support was added only in 5.0?

Reply:
I just updated the packages. I am not sure what is the default behavior but it seems to me the changes here look more consistent with what I think `black` will do. For example,
```
        params = {
            "eps_g": {},
            "eps_i": {},
        }
```
should be in 1 line
```
params = {"eps_g": {}, "eps_i": {}}
```
if it fits into an 88-length (black default line length) line

Reply:
Hmm that's weird. I especially don't understand why linting (`black --check .`) passed both before and after your black changes https://github.com/pyro-ppl/funsor/pull/466/commits/e7dc9d1db7637ed29c9ea8341a987360be607ccd Does `make lint` pass on your system?

Reply:
Yeah, you are right. `black --check .` passed now, and `make format` does not change files anymore. It is weird... I guess black supports two formatting rules for parentheses and based on "cache" or something like that, black will behave differently... Anyway, I will revert unrelated changes because `make format` seems to work now in my system.